CHANGELOG.md

```diff
@@ -7,10 +7,22 @@ since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.or
 
 ## Unreleased
 
+## Dangerzone 0.6.1
+
 ### Added
 
 - Platform support: Ubuntu 24.04 and Fedora 40 ([issue #762](https://github.com/freedomofpress/dangerzone/issues/762))
 
+### Fixed
+
+- Fix OCR bug in Qubes Fedora 38 templates ([issue #737](https://github.com/freedomofpress/dangerzone/issues/737))
+- Development: Get debug logs from the second stage of the conversion (pixels to PDF) ([#676](https://github.com/freedomofpress/dangerzone/pull/676))
```

Comment:
Whoops, my bad. I was referring to https://github.com/freedomofpress/dangerzone/pull/742 actually, and specifically commit a31f3370d0c340080d669fe2fda84807ba9aeb52.

Now that I think of it, it seems kinda out of place. Since it's a dev facing change, I can remove it to avoid confusion. What do you think?

Reply:
Done in f59c368.